EventFilter/Utilities/plugins/TcdsRawToDigi.cc

```diff
@@ -87,20 +88,25 @@ void TcdsRawToDigi::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
     iEvent.getByToken(dataToken_,rawdata);
 
     int nibble=-99;
+    int triggerCount = -99;
     if( rawdata.isValid() ) {
         const FEDRawData& tcdsData = rawdata->FEDData(FEDNumbering::MINTCDSuTCAFEDID);
         if(tcdsData.size()>0){
             evf::evtn::TCDSRecord tcdsRecord(tcdsData.data());
             nibble = (int)tcdsRecord.getHeader().getData().header.nibble;
+            triggerCount = (int)tcdsRecord.getHeader().getData().header.triggerCount;
```

Comment:
if negative numbers should remain useful, I suppose int64_t should still work, since the normal trigger count will not go over 50 bits.

It's still worth to make a proper masking of the high bit (sign).